src/qaqc/01-organize_raw_data.Rmd

```diff
@@ -250,9 +253,11 @@ Add flagging functions for each df in all_data_summary_list
 # This function will be called inside of add_x_flags() functions (ex. add_field_flags())
 # This function will be used to simply add flags into the flag column
 add_flag <- function(df, condition_arg, description_arg) {
+  # adding the flag
   df <- df %>% mutate(flag = case_when(
     {{condition_arg}} ~ if_else(is.na(flag), paste(description_arg), paste(flag, description_arg, sep = ";\n")),
-    TRUE ~ flag)) # <- KW question: what is this part of your code doing? 
+    TRUE ~ flag)) 
+  
   return(df)
```

Comment:
as a general note, per {tidyverse} style guide, you only need to use `return()` if it is an 'early return' in a function. 
```suggestion
 df
```